renderapi/transform/leaf/affine_models.py

```diff
@@ -309,16 +303,48 @@ def inverse_tform(self, points):
         pt = np.dot(np.linalg.inv(self.M), points.T).T
         return self.convert_points_vector_to_array(pt, Nd)
 
+    def calc_properties(self):
```

Comment:
if you make this a property too won't you save on caching the result?

Reply:
you're right, it just seemed like small beans...